frontend_server/lib/server.dart

```diff
@@ -298,9 +291,11 @@ Future<int> starter(
           boundaryKey = string.substring(RECOMPILE_INSTRUCTION_SPACE.length);
           state = _State.RECOMPILE_LIST;
         } else if (string == 'accept') {
-          compiler.acceptLastDelta();
+          // TODO(vegorov) remove this command from the protocol
```

Comment:
I don't understand why this comment says remove this from the protocol, if we don't delete the methods acceptLastDelta, reject etc. and instead just made them nops this code would not have to change,